contracts/round/RoundImplementation.sol

```diff
@@ -253,10 +260,14 @@ contract RoundImplementation is IRoundImplementation, AccessControlEnumerable, I
     token = _token;
 
     // Invoke init on voting contract
-    votingStrategy.init();
+    if (address(votingStrategy) != address(0)) votingStrategy.init();
 
     // Invoke init on payout contract
-    payoutStrategy.init();
+    if (_payoutEncodedParameters.length > 0) {
+      IPayoutStrategyInitializable(payable(payoutStrategy)).init(_payoutEncodedParameters);
+    } else {
+      payoutStrategy.init();
+    }
```

Comment:
What I'd recommend here instead would be to always pass data into init and update the merkle payout strategy to accept the bytes but it doesn't use them. Avoid the if check